docs/docs/services/qix-engine/access-control.md

```diff
@@ -84,347 +140,378 @@ Example JWT:
 
-```c
-(resource.org = "UK") && (user.name = "John Doe")
+### Actions
 
-// Same as previous, but using "and" notation instead of "&&":
-(resource.org = "UK") and (user.name = "John Doe")
-```
+An _Action_ is what operation a user wants to perform on a resource in the Qlik Associative Engine. The actions are:
 
-### EQUAL
+| Action | Description |
+| -------| ----------- |
+| `create` | Create a resource. |
+| `read` | Read a resource. |
+| `update` | Update a resource. |
+| `reload`| Reload a document. |
+| `delete` | Delete a resource. |
+| `import` | Import a document. |
+| `export` | Export a document. |
+| `export data` | Export data from an object. |
 
-This operator is case insensitive and returns `true` if the
-compared expressions are equal. If a list is used, only one
-value needs to match.
+#### The `_actions` Attribute
 
-**Syntax**
+In rule expressions the `resource` attribute `_actions` has special semantics:
 
-```c
-(EXPRESSION) = (EXPRESSION)
-```
+* `resource._actions` shall always be used as left operand with the `=` operator. Other usage is undefined behavior.
+* The expression `resource._actions = (EXPRESSION)` always evaluates to `true`.
+* The expression `resource._actions = (EXPRESSION)` has the "side effect" of accumulating the actions given by the
+  right operand which shall be a single action or a list of actions to be granted.
+
+Hence, omitting `resource._actions = (EXPRESSION)` from a rule would not accumulate any granted actions at all.
+
+It is a good practice to write all rules so that the final expression in the rule involves `resource._actions` to
+define for which actions the access is granted if all other preceeding parts of the rule evaluate to `true`.
 
 **Examples**
 
-Given that `org` is `"uk"` in the access request.
+Given that `user.country` is `"uk"`, the following rule evaluates to `true` and actions granted to users from UK
+are `read` and `update`:
 
-```c
-// Evaluates to `true` because the operator is case insensitive:
-resource.org = "UK"
+```py
+user.country = "uk" and resource._actions = {"read", "update"}
+```
 
-// Evaluates to `true`:
-resource.org = "uk"
+Note that granted actions accumulate in order of rules evaluated. Thus
 
-// Evaluates to `false` because the values are different:
-resource.org = "United Kingdom"
+```py
+user.country = "uk" and resource._actions = {"read", "update"}
+user.roles = {"developer"} and resource._actions = {"create"}
 ```
 
-### LIKE
+first grants users from UK `read` and `update` access. If the user role is `developer`, `create` access is granted.
+Since actions are accumulated, developers from UK are granted access for actions `read`, `update`, and `create`.
 
-The security rules support the regular expression operator "like".
-This operator is case insensitive.
+!!! Note
+    This is what we might want, but care must be taken here. If the role is `developer` but the user is _not_ from UK,
```

Comment:
You are not referencing `we` or `us` anywhere else, this note also feels a little repetitive, rewrite?